docs/semgrep-app/notifications.md

```diff
@@ -59,8 +59,168 @@ To view, add, remove, disable, or enable your saved channels:
 
 ### Slack
 
+<!-- When referring to the Slack integration, let's consistently call it the "Semgrep Slack app" (app in lower case). This is consistent with the terminology used by Slack (they refer to these integrations as apps in their product pages. It also makes sure that the user knows what "app" we're referring to.
+
+Prefer "Semgrep Slack app" over "Semgrep Slack integration" or "Slack integration" or any other variant. -->
+
 <ProcedureIntegrateSlack />
 
+#### Slack permissions 
+
+The following table describes the purpose for each permission required to use the Semgrep Slack app. 
+
+<table>
+  <tr>
+   <td><strong>Permission</strong>
+   </td>
+   <td><strong>Slack description</strong>
```

Comment:
FYI Normally, we don't include descriptions for parts of another product because the product can change without notice. Instead, we link to those docs. 

HOWEVER, I am suggesting we keep the Slack description because 

1. It adds useful context (plus we don't want the user to have to alt-tab constantly.)
2. We actually WILL know when ANY changes on Slack's end happens, because our integration is dependent on Slack's behavior. In short, we will always know when we have to update these doc descriptions.

This is just an FYI explaining why I think this is useful (thank you to Vivek for providing them.)

Reply:
Good idea! If we get easily notified about the changes and others will help us update this document time from time to, this is a great thing for our users!